os/board/rtl8721csm/src/component/common/mbed/targets/hal/rtl8721d/flash_api.c

```diff
@@ -214,7 +211,6 @@ int  flash_read_word(flash_t *obj, u32 address, u32 * data)
 	}else{

```

Comment:
please add blank each side of else

Reply:
Please ignore this comment, you don't need to apply our coding rules for chip vendor codes.